CHANGELOG.md

```diff
@@ -1,36 +1,52 @@
 * Added support to project Lombok. (#502)
-* Deprecated methods: Realm.getInstance(Context). Use Realm.getInstance(RealmConfiguration) or Realm.getDefaultInstance() instead.
-* Deprecated methods: Realm.getTable(Class) which was public because of the old migration API. Use Realm.getSchema() or DynamicRealm.getSchema() instead.
-* Deprecated Realm.executeTransaction(Transaction, Callback) and replaced it with Realm.executeTransactionAsync(Transaction), Realm.executeTransactionAsync(Transaction, OnSuccess), Realm.executeTransactionAsync(Transaction, OnError) and Realm.executeTransactionAsync(Transaction, OnSuccess, OnError).
-* Fixed an error occurring during test and connectedCheck of unit test example (#1934).
-* Fixed bug in jsonExample (#2092).
-* Fixed bug when multiple calls of RealmResults.distinct() causes to return wrong results (#2198).
-* Fixed bug when calling DynamicRealmObject.setList() with RealmList<DynamicRealmObject> (#2368).
-* Fixed RealmChangeListeners not triggering correctly if findFirstAsync() didn't find any object. findFirstAsync() Observables now also correctly call onNext when the query completes in that case (#2200). (thank you @grahamsmith)
 * Added RealmQuery.isNotEmpty() (#2025). (Thank you @stk1m1)
 * Added Realm.deleteAll() and RealmList.deleteAllFromRealm() (#1560).
 * Added RealmQuery.distinct() and RealmResults.distinct() (#1568).
 * Added RealmQuery.distinctAsync() and RealmResults.distinctAsync() (#2118).
 * Improved .so loading by using [ReLinker](https://github.com/KeepSafe/ReLinker).
 * Improved performance of RealmList#contains() (#897).
-* RealmResults.size() now returns Integer.MAX_VALUE when actual size is greater than Integer.MAX_VALUE (#2129).
-* Removed allowBackup from AndroidManifest (#2307).
 * Added multi-arguments distinct(...) for Realm, DynamicRealm, RealmQuery, and RealmResults (#2284).
 * Allowed "realm" and "row" as a field name of model class (#2255).
-* Updated Realm Core to 0.97.0
-  - Fixed bug when setting a null value to trigger RealmChangeListener (#2366).
-  - Fixed bug to prevent throwing BadVersionException (#2391).
+* RealmResults.size() now returns Integer.MAX_VALUE when actual size is greater than Integer.MAX_VALUE (#2129).
+* Removed allowBackup from AndroidManifest (#2307).
+
+### Bug fixes
+
+* Fixed an error occurring during test and connectedCheck of unit test example (#1934).
+* Fixed bug in jsonExample (#2092).
+* Fixed bug when multiple calls of RealmResults.distinct() causes to return wrong results (#2198).
+* Fixed bug when calling DynamicRealmObject.setList() with RealmList<DynamicRealmObject> (#2368).
+* Fixed RealmChangeListeners not triggering correctly if findFirstAsync() didn't find any object. findFirstAsync() Observables now also correctly call onNext when the query completes in that case (#2200). (thank you @grahamsmith)
+
+### Updated Realm Core to 0.97.0
```

Comment:
When we upgrade Realm Core, we could mention it before any subsection, and then simply add the items to either "Enhancements" or "Bug fixes".